features.md

```diff
@@ -79,7 +79,7 @@ names will be added to this list as they are actually defined.
 <a name="fn4">[4]</a>: To enable this, use the Chrome command line flag
 `--enable-blink-features=Serial`.
 
-<a name="fn5">[5]</a>: The earlier version of this feature ([`lazyload`](policies/lazyload.md)) is available behind a flag in Chrome<sup>[1](#fn1)</sup>.
+<a name="fn5">[5]</a>: The earlier version of this feature ([`lazyload`](https://www.chromestatus.com/feature/5641405942726656)) is available behind a flag in Chrome<sup>[1](#fn1)</sup>.
```

Comment:
Does it make sense for this to link to https://www.chromestatus.com/feature/564140594272665 since lazyload.md?

Reply:
That makes sense to me -- I'd update the commit message to note that that's being changed in this PR as well, though.

Reply:
@clelland Because it isn't possible edit commit messages on github.com I had to use Git for the first time. I reverted the commit and added back changes with a better commit message, do you deem the changes acceptable?

Reply:
Oh, that's annoying :( Sorry about that. Yes, this is fine, and thanks!